extensions/ql-vscode/package.json

```diff
@@ -970,6 +983,10 @@
           "command": "codeQL.chooseDatabaseLgtm",
           "when": "config.codeQL.canary"
         },
+        {
+          "command": "codeQLDatabases.openConfigFile",
```

Comment:
Sorry for the ignorance, but why do we need to do this?

Reply:
This stops the command showing up in the command palette, so it's only accessible from the Databases panel. 

I guess we could make it more widely visible later, but I didn't see any reason to do that yet 🤔

Reply:
Oh ofc! Sorry I forget that these commands are automatically added to the command pallet.